app/src/modules/content/routes/item.vue

```diff
@@ -173,6 +176,15 @@ const {
 	revisionsAllowed,
 } = usePermissions(collection, item, isNew);
 
+const updateVersionsAllowed = computed<boolean>(() => hasPermission('directus_versions', 'update'));
+
+const isFormDisabled = computed(() => {
+	if (isNew.value) return false;
+	if (updateAllowed.value) return false;
+	if (updateVersionsAllowed.value && currentVersion.value !== null) return false;
```

Comment:
```suggestion
If not used anywhere else, I'd include it directly in `isFormDisabled` (li’l perf optimization)

```suggestion
const isFormDisabled = computed(() => {
	if (isNew.value) return false;
	if (updateAllowed.value) return false;
	const updateVersionsAllowed = hasPermission('directus_versions', 'update');
	if (currentVersion.value !== null && updateVersionsAllowed) return false;
```